db/migrate/20230613153019_add_foreign_key_constraint_to_application_id_on_oauth_access_tokens.rb

```diff
@@ -0,0 +1,14 @@
+class AddForeignKeyConstraintToApplicationIdOnOauthAccessTokens < ActiveRecord::Migration[7.0]
+  def change
+    Doorkeeper::AccessToken.all.select { |at| at.application.nil? }.each do |access_token|
```

Comment:
I ran `Doorkeeper::AccessToken.all.select { |at| at.application.nil? }.count` in a production console to check this wouldn't take too long. It returned `0` as expected in that environment and took around 15s, so I think it'll be fine to do this in the migration.

Reply:
Thanks, @chrislo!